meteor/server/api/showStyles.ts

```diff
@@ -132,18 +176,48 @@ export async function removeShowStyleVariant(
 	await ShowStyleVariants.removeAsync(showStyleVariant._id)
 }
 
+export async function reorderAllShowStyleVariants(
+	context: MethodContext,
+	showStyleBaseId: ShowStyleBaseId,
+	orderedVariants: ShowStyleVariant[]
```

Comment:
No need to send the full data set over the wire here, sinve only the `._id` is used.
This should be refactored to `ShowStyleVariantId[]` instead.